lib/block-supports/border.php

```diff
@@ -13,7 +13,7 @@
  */
 function gutenberg_register_border_support( $block_type ) {
 	// Determine if any border related features are supported.
-	$has_border_support       = gutenberg_block_has_support( $block_type, array( '__experimentalBorder' ) );
+	$has_border_support       = block_has_support( $block_type, array( '__experimentalBorder' ) );
```

Comment:
https://developer.wordpress.org/reference/functions/block_has_support/ was introduced in WordPress 5.8 and that is the minimum supported version for the plugin.